<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <rosparam file="$(find mouse_description)/config/mouse.yaml"
            command="load"/>

  <node name="robot_state_publisher_mouseA" pkg="robot_state_publisher" type="robot_state_publisher"
        respawn="false" output="screen">
            <param name="publish_frequency" type="double" value="30.0" />
            <param name="ignore_timestamp" type="bool" value="true" />
            <param name="tf_prefix" type="string" value="mouseA" />
            <remap from="/joint_states" to="/mouseA/joint_states" />
        </node>


  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
            args="--namespace=/mouseA
                      joint_state_controller
                      left_diff_drive_controller
                      right_diff_drive_controller">
  </node>

  <node name="mouseA_odom_pubish_node"
        pkg="mouse_description"
        type="get_gazebo_model_odometry.py" respawn="false"
        output="screen" args="mouseA">
  </node>

  <node name="mouseA_decision_node"
        pkg="mouse_description"
        type="mouse_movement.py" respawn="false"
        output="screen" args="">
  </node>

</launch>
